examples/web-wagmi/src/profiles/UseFollowAndUnfollow.tsx

```diff
@@ -59,7 +63,7 @@ type UseFollowInnerProps = {
 };
 
 function UseFollowInner({ activeProfile }: UseFollowInnerProps) {
-  const { data, error, loading } = useExploreProfiles({ limit: 50 });
+  const { data, error, loading } = useExploreProfiles({ limit: 50, observerId: activeProfile.id });
```

Comment:
`observerId` is not longer needed to be explicit if you want it to be about the Active Profile. Did you find issues with this working as expected?

Reply:
okay great, thanks. no issues - this was leftover from prior implementation